spec/lib/object_serializer_class_methods_spec.rb

```diff
@@ -418,6 +418,25 @@ class AreaSerializer
 
       after do
         MovieSerializer.attributes_to_serialize.delete(:released_in_year)
+        MovieSerializer.attributes_to_serialize.delete(:name)
```

Comment:
This is here because the old spec was not cleaned up (it set up `name` attribute in `before_hook` but did not remove it again - it has nothing to do with the fix.